Shell/CmdFullWrite.cpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include "CmdFullWrite.h"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "SSDInterface.h"
+
+CommandFullWrite::CommandFullWrite(SSDInterface* ssdInterface)
+    : ssd(ssdInterface) {}
+
+bool CommandFullWrite::Call(std::vector<std::string> program) {
```

Comment:
본 함수에서 program 값을 변경하지 않으므로 명시적으로 `const &`로 선언하면 좋을 것 같습니다.

```suggestion
bool CommandFullWrite::Call(const std::vector<std::string>& program) {
```